src/test/resources/org/prebid/server/it/openrtb2/rubicon_appnexus/test-cache-rubicon-appnexus-request.json

```diff
@@ -17,7 +18,8 @@
         "adm": "adm2",
         "crid": "crid2",
         "w": 300,
-        "h": 600
+        "h": 600,
+        "wurl": "http://localhost:8000/event?t=win&b=BIDID&a=5001&f=i"
```

Comment:
Please clarify with Ruslan, if this tests will not fail on another environment bc of `http://localhost:8000/`

Reply:
That's fine. Will refactor in future.